kubernetes/windows/main.ps1

```diff
@@ -695,6 +716,12 @@ function Start-Fluent-Telegraf {
 
     $containerRuntime = Get-ContainerRuntime
 
+    $monitorKubernetesPods = [System.Environment]::GetEnvironmentVariable('TELEMETRY_CUSTOM_PROM_MONITOR_PODS')
```

Comment:
isn't this also already happening below as well when we check SIDECAR_SCRAPING_ENABLED

Reply:
> isn't this also already happening below as well when we check SIDECAR_SCRAPING_ENABLED
 didnt understand your comment and lets chat. SIDECAR_SCRAPING_ENABLED being set to true by default in toggle yaml(aks) and for non-aks, values.yaml to true and hence its no-op.